src/libclient/canvas/userlist.cpp

```diff
@@ -249,13 +249,21 @@ QVector<uint8_t> UserListModel::trustedList() const
 	return ids;
 }
 
-User UserListModel::getUserById(int id) const
+User UserListModel::getUserById(int id, bool *outFound) const
```

Comment:
`std::optional`

Reply:
Ah right, we got C++17 now. Changed that. It doesn't end up being shorter, but that's why they made C++ a postincrement I guess.